packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.js

```diff
@@ -0,0 +1,94 @@
+import { LightningElement, api } from 'lwc';
+
+/**
+ * The `descriptionStrengthIndicator` component is a dynamic indicator that shows the user if their case description has enough details so far. It also give hints to the user as to what information they show include in the description. 
+ * @example
+ * <c-description-strength-indicator></c-description-strength-indicator>
+ */
+export default class DescriptionStrengthIndicator extends LightningElement {
+  /**
+   * The label to be shown to the user.
+   * @api
+   * @type {string}
+   * @defaultValue `'Don’t know what to write?'`
+   */
+  @api helpLabel = 'Don’t know what to write?';
+
+  /**
+   * The initial message to be shown to the user.
+   * @api
+   * @type {string}
+   * @defaultValue `'Provide details'`
+   */
+  @api initialMesssage = 'Provide details';
+
+  /**
+   * The message to be shown to encourage the user to keep going.
+   * @api
+   * @type {string}
+   * @defaultValue `'Provide more details'`
+   */
+  @api keepGoingMessage = 'Provide more details';
+
+  /**
+   * The progress value from where the keepGoingMessage will be shown.
+   * @api
+   * @type {number}
+   * @defaultValue `75`
+   */
+  @api keepGoingThreshold = 75;
+
+  /**
+   * The final message to be shown to the user when the progress indicator is full.
+   * @api
+   * @type {string}
+   * @defaultValue `'Thank you!'`
+   */
+  @api finalMessage = "Thank you!";
```

Comment:
Do we need to create localized label for those strings ?

Reply:
Yes I'm adding them 👌